src/components/budget/budget.service.ts

```diff
@@ -215,10 +215,10 @@ export class BudgetService {
     }
 
     if (!result) {
-      this.logger.error(`Could not find budget:  `, {
-        id,
-        userId: session.userId,
-      });
+      // this.logger.error(`Could not find budget:  `, {
+      //   id,
+      //   userId: session.userId,
+      // });
```

Comment:
This could be left and changed to a warning. Otherwise delete it; don't leave commented out code.
```suggestion
     this.logger.warning(`Could not find budget`, {
        id,
        userId: session.userId,
      });
```

Reply:
Thank you @CarsonF